File: sample_app/src/main/java/uk/co/alt236/btlescan/ui/details/recyclerview/model/TextItem.java
package uk.co.alt236.btlescan.ui.details.recyclerview.model;

import uk.co.alt236.btlescan.ui.common.recyclerview.RecyclerViewItem;

public class TextItem implements RecyclerViewItem {
    private final CharSequence mText;

    public TextItem(CharSequence text) {
        mText = text;
    }

    public CharSequence getText() {
        return mText;
    }
}

File: sample_app/src/main/java/uk/co/alt236/btlescan/ui/details/recyclerview/model/TextItem.kt
package uk.co.alt236.btlescan.ui.details.recyclerview.model

import uk.co.alt236.btlescan.app.ui.view.details.model.TextItem
import uk.co.alt236.btlescan.app.ui.view.recyclerview.RecyclerViewItem

class TextItem(
    override val text: CharSequence,
) : RecyclerViewItem,
    TextItem
